javascript-light-bulb/main.js

```diff
@@ -1,14 +1,12 @@
 var $cont = document.querySelector('#cont');
-var $bulb = document.querySelector('#bulb');
 
 function onOff(event) {
-  if ($bulb.className === 'bulb-off') {
-    $bulb.className = 'bulb-on';
-    $cont.className = 'cont-on';
+  var contClass = $cont.className;
+  if (contClass === 'off') {
```

Comment:
This is a step in the right direction, but you are still dependent on data from the DOM to determine your course of action. You would probably benefit with a variable that dictates whether the bulb is currently on or not.